packages/@aws-cdk/cfnspec/spec-source/000_CloudFormationResourceSpecification.json

```diff
@@ -33429,7 +33508,7 @@
         "Protocol": {
           "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sns-subscription.html#cfn-sns-protocol",
           "PrimitiveType": "String",
-          "Required": false,
+          "Required": true,
```

Comment:
Uh - that's a dodgy change... Making optional properties required... Wow CFN, not thank you!